third_party/xnnpack.buck.bzl

```diff
@@ -86,7 +86,17 @@ def define_xnnpack(third_party, labels = [], XNNPACK_WINDOWS_AVX512F_ENABLED = F
     fb_xplat_cxx_library(
         name = "operators",
         # srcs have to include HOT_SRCS to be able to build on ARVR
```

Comment:
Please remove this comment if this is no longer the case

Reply:
Ok!

Reply:
Suggest merging now to catch nightly. Follow up to address this comment issue?

Reply:
Ok! yeah!

Reply:
Hmm, but it did not catch the nightly, so perhaps it should be changed now?